apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+export type ComponentCardProps = {
+  /** Component name. */
+  name: string;
+  /** If true a "Implemented" badge will be shown, "Coming Soon" otherwise. */
+  implemented?: boolean;
+};
+
+const props = defineProps<ComponentCardProps>();
+</script>
+
+<template>
+  <div class="card">
+    <article class="card__content">
+      <Badge
+        v-if="props.implemented"
+        class="card__badge card__badge--implemented"
+        text="Implemented"
+      />
+      <Badge v-else class="card__badge" text="Coming Soon" type="info" />
```

Comment:
I mean we will implement those components, right?^^ At least we currently plan to do so.
I think "Planned in the future" sounds more like "If you are lucky we might implement it in 5 years"^^

Reply:
ok, then maybe just "Planned"? I think "soon" is misleading. it might take 9 months or so for the final tier

Reply:
Maybe just "Planned"?
it might take some months until expansion 4 is done, soon sounds a bit misleading ¯\_(ツ)_/¯ 

but it's not that bad, if you don't want to overcomplicate things, maybe the users won't even notice ^^

Reply:
Planned sounds good, I changed it